src/services/schema-export/tests/models/all/java/ChildEmbeddedType.java

```diff
@@ -0,0 +1,14 @@
+// Please note : @LinkingObjects and default values are not represented in the schema and thus will not be part of the generated models
+package your.package.name.here;
+
+import io.realm.RealmObject;
+
+public class ChildEmbeddedType extends RealmObject {
```

Comment:
This should be 
```
@RealmClass(embedded = true)
public class ChildEmbeddedType extends RealmObject {
```

and `io.realm.annotations.RealmClass` should be added to the list of imports.